libs/langgraph/src/prebuilt/react_agent_executor.ts

```diff
@@ -30,14 +31,20 @@ import {
 import { MessagesAnnotation } from "../graph/messages_annotation.js";
 import { ToolNode } from "./tool_node.js";
 import { LangGraphRunnableConfig } from "../pregel/runnable_types.js";
+import { Annotation } from "../graph/annotation.js";
+import { Messages, messagesStateReducer } from "../graph/message.js";
 
+type StructuredResponse =
+  | z.ZodType<Record<string, unknown>>
```

Comment:
Don't think `z.ZodType` belongs in inputs? We are not outputting a zod schema

Let's also type as `Record<string, any>` until we get better typing